src/components/Hero.tsx

```
@@ -1,63 +1,67 @@
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              className="mr-1 inline-block size-5"
+              viewBox="0 0 24 24"
+              fill="currentColor"
+            >
+              <path d="M0 24V0h24v24H0zM6.951 5.896l4.112 7.708v5.064h1.583v-4.972l4.148-7.799h-1.749l-2.457 4.875c-.372.745-.688 1.434-.688 1.434s-.297-.708-.651-1.434L8.831 5.896h-1.88z" />
+            </svg>
+            <span>Featured on Hacker News</span>
+          </a>
+          <div dangerouslySetInnerHTML={{ __html: productHuntBadgeHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)